bindings.cpp

```diff
@@ -1700,7 +1700,8 @@ bool DeRestPluginPrivate::sendConfigureReportingRequest(BindingTask &bt)
         rq2.maxInterval = 300;
         if (sensor && (sensor->modelId() == QLatin1String("SmartPlug") || // Heiman
                        sensor->modelId() == QLatin1String("902010/25") || // Bitron
-                       sensor->modelId() == QLatin1String("SKHMP30-I1") ||// GS smart plug
+                       sensor->modelId() == QLatin1String("SKHMP30-I1") || // GS smart plug
+                       sensor->modelId() == QLatin1String("SMRZB-1") || // Develco smart cable
```

Comment:
This one is not necessary, since it already reports in W as I recall

Reply:
Done.